specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-03-01-preview/AutomationRules.json

```diff
@@ -217,6 +217,64 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/entities/{entityIdentifier}/runPlaybook": {
+      "post": {
+        "tags": [
+          "manualTrigger"
+        ],
+        "description": "Triggers playbook on a specific entity.",
+        "operationId": "Entities_RunPlaybook",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "in": "path",
+            "name": "entityIdentifier",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "requestBody",
+            "schema": {
+              "$ref": "#/definitions/EntityManualTriggerRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
```

Comment:
>204 [](http://example.com/codeflow?start=11&length=3)

Why is this 204? If action is completed successfully, it should return 200

Reply:
This action is calling LogicApps API to trigger a playbook on an entity. It returns 204 since we know that the invocation was "accepted" but we don't know whether it was completed successfully.
The response of this endpoint aligns with the response code in another similar endpoint that we have - "/incidents/{incidentID}/runPlaybook" that returns 204 for the same reason and it has existed for a long time now.